main/src/vtr_gui/vtr_gui/vtr-gui/src/components/graph/GraphMap.js

```diff
@@ -143,6 +145,8 @@ class GraphMap extends React.Component {
       move_robot_vertex: { lng: 0, lat: 0, id: -1 },
       // map center
       map_center: {lat: 43.78220, lng: -79.4661},
+      /// whether the path selection will be the whole path (for annotation)
+      annotateFull: false,
```

Comment:
If it's not to hard to refactor, it would be preferable to use `snake_case` instead of `camelCase` to have consistent style